data/jsons/dependencies.json

```diff
@@ -0,0 +1,123 @@
+{
+    "dependencies": [
+	{
+	    "dependency": "check",
+	    "type": "pkg-config",
+	    "pkgname": "check"
```

Comment:
Yep. Besides the version checking think, r+ for the series. I'm working on top of it for a Galileo build and it's fine. Let's merge this ASAP, because I already have patches on top of it.